### ISSUE TITLE
Classification for Allow header field (Trac #105)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#105]
Currently, "Allow" is defined as an entity-header, not a response-header. Its definition says 

  The Allow entity-header field lists the set of methods supported by the resource identified by the Request-URI." 

However, Allow seems more applicable to the resource, rather than the entity, and isn't useful in a request.

Migrated from https://trac.ietf.org/ticket/105
```json
{
    "status": "closed",
    "changetime": "2012-07-14T13:56:28",
    "_ts": "1342274188114811",
    "description": "Currently, \"Allow\" is defined as an entity-header, not a response-header. Its definition says \n\n  The Allow entity-header field lists the set of methods supported by the resource identified by the Request-URI.\" \n\nHowever, Allow seems more applicable to the resource, rather than the entity, and isn't useful in a request.",
    "reporter": "mnot@pobox.com",
    "cc": "",
    "resolution": "fixed",
    "time": "2008-02-28T00:56:12",
    "component": "p2-semantics",
    "summary": "Classification for Allow header field",
    "priority": "normal",
    "keywords": "",
    "milestone": "03",
    "owner": "julian.reschke@gmx.de",
    "type": "design",
    "severity": "Active WG Document"
}
```

**Comment by mnot** [comment]
Trac update at 20080303T02:06:59: mnot@pobox.com commented:

> Proposal:
> 
> Change the classification of Allow to a response header, removing the following text;
> 
>       The Allow header field MAY be provided with a PUT request to
>       recommend the methods to be supported by the new or modified
>       resource. The server is not required to support these methods and
>       SHOULD include an Allow header in the response giving the actual
>       supported methods.
> 
> (or moving to an appendix and marking as deprecated behaviour)

**Comment by mnot** [comment]
Trac update at 20080311T04:38:11: 
* mnot@pobox.com commented:

> Closed with proposal above.


* mnot@pobox.com changed milestone from "unassigned" to "03"
* mnot@pobox.com changed resolution from "" to "fixed"
* mnot@pobox.com changed status from "new" to "closed"

**Comment by mnot** [comment]
Trac update at 20080311T04:46:42: 
* mnot@pobox.com changed resolution from "fixed" to ""
* mnot@pobox.com changed status from "closed" to "reopened"

**Comment by mnot** [comment]
Trac update at 20080311T08:37:34: 
* julian.reschke@gmx.de commented:

> This really affects P2. P3 is only affected in that "Allow" does not appear in the list of entity headers anymore.
> 


* julian.reschke@gmx.de changed component from "payload" to "semantics"
* julian.reschke@gmx.de changed owner from "" to "julian.reschke@gmx.de"
* julian.reschke@gmx.de changed status from "reopened" to "new"

**Comment by mnot** [comment]
Trac update at 20080311T08:43:49: 
* julian.reschke@gmx.de commented:

> Fixed in [232]:
> 
> Resolve #105: Reclassify Allow header as response header (closes #105).


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "new" to "closed"

**Comment by mnot** [comment]
Trac update at 20120714T13:56:28: 
* julian.reschke@gmx.de changed priority from "" to "normal"
* julian.reschke@gmx.de changed severity from "" to "Active WG Document"
* julian.reschke@gmx.de changed summary from "Classification for Allow header" to "Classification for Allow header field"
